[2023-01-28 10:53:36 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-01-28 10:53:36 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2023-01-28 10:53:36 +0000] [1] [INFO] Using worker: sync
[2023-01-28 10:53:36 +0000] [7] [INFO] Booting worker with pid: 7
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-01-28 11:07:04 +0000] [1] [INFO] Handling signal: term
[2023-01-28 11:07:04 +0000] [7] [INFO] Worker exiting (pid: 7)
[2023-01-28 11:07:06 +0000] [1] [INFO] Shutting down: Master
[2023-01-28 11:07:08 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-01-28 11:07:08 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2023-01-28 11:07:08 +0000] [1] [INFO] Using worker: sync
[2023-01-28 11:07:08 +0000] [7] [INFO] Booting worker with pid: 7
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-01-28 11:23:20 +0000] [1] [INFO] Handling signal: term
[2023-01-28 11:23:20 +0000] [7] [INFO] Worker exiting (pid: 7)
[2023-01-28 11:23:22 +0000] [1] [INFO] Shutting down: Master
[2023-01-28 11:23:30 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-01-28 11:23:30 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2023-01-28 11:23:30 +0000] [1] [INFO] Using worker: sync
[2023-01-28 11:23:30 +0000] [7] [INFO] Booting worker with pid: 7
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-01-28 11:39:00 +0000] [1] [INFO] Handling signal: term
[2023-01-28 11:39:00 +0000] [7] [INFO] Worker exiting (pid: 7)
[2023-01-28 11:39:05 +0000] [1] [INFO] Shutting down: Master
[2023-01-28 11:39:16 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-01-28 11:39:16 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2023-01-28 11:39:16 +0000] [1] [INFO] Using worker: sync
[2023-01-28 11:39:16 +0000] [7] [INFO] Booting worker with pid: 7
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-01-28 11:39:45 +0000] [1] [INFO] Handling signal: term
[2023-01-28 11:39:45 +0000] [7] [INFO] Worker exiting (pid: 7)
[2023-01-28 11:39:46 +0000] [1] [INFO] Shutting down: Master
[2023-01-28 11:50:09 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-01-28 11:50:09 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2023-01-28 11:50:09 +0000] [1] [INFO] Using worker: sync
[2023-01-28 11:50:09 +0000] [7] [INFO] Booting worker with pid: 7
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-01-28 11:50:21,508] ERROR in app: Exception on /api/tts [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 471, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 500, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/usr/lib/python3.10/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.10/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.10/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.10/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
[2023-01-28 11:59:15 +0000] [1] [INFO] Handling signal: term
[2023-01-28 11:59:15 +0000] [7] [INFO] Worker exiting (pid: 7)
[2023-01-28 11:59:15 +0000] [1] [INFO] Shutting down: Master
[2023-01-29 03:40:14 +0000] [1] [INFO] Starting gunicorn 20.1.0
[2023-01-29 03:40:14 +0000] [1] [INFO] Listening at: http://0.0.0.0:8000 (1)
[2023-01-29 03:40:14 +0000] [1] [INFO] Using worker: sync
[2023-01-29 03:40:14 +0000] [7] [INFO] Booting worker with pid: 7
'FLASK_ENV' is deprecated and will not be used in Flask 2.3. Use 'FLASK_DEBUG' instead.
[2023-01-29 03:40:50,355] ERROR in app: Exception on /api/tts [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 471, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 500, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/usr/lib/python3.10/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.10/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.10/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.10/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
[2023-01-29 03:41:20,981] ERROR in app: Exception on /api/tts [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 471, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 500, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/usr/lib/python3.10/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.10/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.10/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.10/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
[2023-01-29 03:41:33,133] ERROR in app: Exception on /api/tts [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 471, in wrapper
    return self.make_response(data, code, headers=headers)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/__init__.py", line 500, in make_response
    resp = self.representations[mediatype](data, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/flask_restful/representations/json.py", line 21, in output_json
    dumped = dumps(data, **settings) + "\n"
  File "/usr/lib/python3.10/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.10/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.10/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.10/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type bytes is not JSON serializable
[2023-01-29 03:43:29 +0000] [1] [INFO] Handling signal: term
[2023-01-29 03:43:29 +0000] [7] [INFO] Worker exiting (pid: 7)
[2023-01-29 03:43:30 +0000] [1] [INFO] Shutting down: Master
